test/integration/visitor_can_view_items_test.rb

```diff
@@ -14,4 +14,33 @@ class VisitorCanViewItemsTest < ActionDispatch::IntegrationTest
       assert page.has_content?("Dang Coconut")
     end
   end
+
+  test "visitor can view items by category" do
+
+    category_1 = OilType.create(name: "Lard")
+    category_2 = OilType.create(name: "Coconut Oil")
+
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_type_id: category_1.id)
+    item_2 = Chip.create(name: "Trader Joe's BBQ", price: 15, description: "I'd trade slota for these!", oil_type_id: category_2.id)
+    item_3 = Chip.create(name: "Dang Coconut", price: 17, description: "Dang, these are good", oil_type_id: category_2.id)
```

Comment:
Useless assignment to variable - `item_3`. Did you mean `item_1`?<br>Line is too long. [122/80]